### PR TITLE
30JUN20 commit for CyberArk Sentinel integration

### PR DESCRIPTION
**Pull request by JimmyJamCABD**
I do not understand what is going on with the Playbooks/Get-MDATPVulnerabilities/report_template.docx  I have added it to my local repo and no matter what I do it picks up on it missing, I do not understand why.

### CODE REVIEW

**Comment by JimmyJamCABD** [comment]
Cleared JSON Validation errors

**Comment by JimmyJamCABD** [comment]
One more try

**Comment by JimmyJamCABD** [comment]
Reopening to attempt to navigate past cancelled pipeline run

File: DataConnectors/CyberArk Data Connector.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+	"id": "CyberArk",
```
Comment (preetikr) [inline DataConnectors/CyberArk Data Connector.json:2]: All the issues reported in the earlier PR #764 for the data connector is not incorporated - please incorporate those.

File: Workbooks/CyberArkEPV.json [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Conjur Vault syncs"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/e600a6ef-1740-4ee6-a6ba-3671a70986c3/resourcegroups/securitycenter/providers/microsoft.operationalinsights/workspaces/cyberark-devel"
```
Comment (preetikr) [inline Workbooks/CyberArkEPV.json:194]: Remove the subscription ID from the workbook as this will be a publicly shared workbook.

**Pull request closed without merging**